File: 240416/길이 단위 변환하기/change-length-unit.py
a = 9.2 #피트(ft) 
b = 1.3 #마일(mi)

FT_TO_CM = 30.48 # cm
MI_TO_CM = 160934 #cm

a_convert_cm = a * FT_TO_CM
b_convert_cm = b * MI_TO_CM

print("%0.1fft = %0.1fcm" %(a,a_convert_cm))
print("%0.1fmi = %0.1fcm" %(b,b_convert_cm))

File: 240429/길이 단위 변환하기/change-length-unit.cpp
#include <iostream>
using namespace std;
int main() {
    double a = 9.2;
    double b = 1.3;
    double ftToCm = 30.48;
    double miToCm = 160934; 

    cout << fixed;    
    cout.precision(1);
    cout << a << "ft = " << a * ftToCm << "cm" << endl;
    cout << b << "mi = " << b * miToCm << "cm" << endl;
    return 0;
}